src/werkzeug/utils.py

```diff
@@ -75,22 +77,22 @@ def value(self):
 
     def __init__(
         self,
-        fget: t.Callable[[t.Any], t.Any],
+        fget: t.Callable[[t.Any], _T],
         name: t.Optional[str] = None,
         doc: t.Optional[str] = None,
     ) -> None:
         super().__init__(fget, doc=doc)
         self.__name__ = name or fget.__name__
         self.__module__ = fget.__module__
 
-    def __set__(self, obj: object, value: t.Any) -> None:
+    def __set__(self, obj: object, value: _T) -> None:
         obj.__dict__[self.__name__] = value
 
-    def __get__(self, obj: object, type: type = None) -> t.Any:  # type: ignore
+    def __get__(self, obj: object, type: type = None) -> _T:  # type: ignore
         if obj is None:
-            return self
+            return self  # type: ignore
 
-        value = obj.__dict__.get(self.__name__, _missing)
+        value = obj.__dict__.get(self.__name__, _missing)  # type: _T
```

Comment:
I'd like to avoid type comments, was it necessary here?

Reply:
Without it, `value` is inferred to be `Any` and I got a mypy warning about not returning `Any`.